src/engine/bank01.asm

```diff
@@ -5745,7 +5745,7 @@ Func_64b0: ; 64b0 (1:64b0)
 	call LoadDuelCheckPokemonScreenTiles
 	ld de, wDuelTempList
 	call SetListPointer
-	ld a, $ef
+	ld a, LOW(wPlayerNumberOfPokemonInPlayArea)
```

Comment:
LOW was used already to define the constant that you're looking for. DUELVARS_NUMBER_OF_POKEMON_IN_PLAY_AREA is the more correct value to put here, in form with the other calls to GetTurnDuelistVariable.

Reply:
Changed LOW(wPlayerNumberOfPokemonInPlayArea) to DUELVARS_NUMBER_OF_POKEMON_IN_PLAY_AREA